src/App/Application.cpp

```diff
@@ -1114,6 +1115,11 @@ Application::TransactionSignaller::~TransactionSignaller() {
     }
 }
 
+int64_t Application::applicationPid()
+{
+    return QCoreApplication::applicationPid();
+}

```

Comment:
If I instead use this I can launch two flatpak instances, would that be sensible?
```suggestion
int64_t Application::applicationPid()
{
    static int64_t randomNumber = []() {
        auto seed = std::chrono::high_resolution_clock::now().time_since_epoch().count();
        std::mt19937 generator(static_cast<unsigned>(seed));
        std::uniform_int_distribution<int> distribution(1, 1000000);
        return distribution(generator);
    }();
    return randomNumber;
}
```

Reply:
I would do this in a second PR, then. Although it should work it still requires some testing.